docs/getting-started/install.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Getting Started
+subtitle: An overview of Athena, including how to download and use it, some basic templates and examples, and more.
+---
+
+# Installation
+
+Installing the Athena Framework is as simple as adding the CSS and JS files to your project.
+
+<hr />
+
+## Minified files
+
+After <a href="{{ "/getting-started/download/" | prepend: site.baseurl }}">downloading and extracting the Athena-Framework,</a> add the CSS, JS and Font files to your project's static assets folder.
```

Comment:
We might want to add a note that the files in /dist/ need to maintain their directory structure wherever they're installed (Athena references a relative path to the font files which will break if fonts are installed up/down another directory)